src/main/java/com/github/nicholasmoser/ToolController.java

```diff
@@ -12,9 +12,13 @@
 import com.github.nicholasmoser.tools.SeqDisassemblerTool;
 import com.github.nicholasmoser.tools.SeqEditorTool;
 import com.github.nicholasmoser.tools.TXG2TPLTool;
+
+import java.awt.*;
```

Comment:
Can you remove this * import and replace it with the specific class used? If two packages have a class of the same name it can cause issues where you think it's using one class but is actually using a different one. Just import the specific class this is using.

IntelliJ actually tries to use * imports by default, and you can avoid this by going to `File->Settings` and then under `Editor->Code Style->Java` and change the class and name count to 999:
![image](https://user-images.githubusercontent.com/4983605/163650470-569ae3e0-b011-4558-8f44-4dae3f3d79e3.png)

Reply:
Actually I'm planning on going through and cleaning up the opcodes, so I can make this change myself